elogia_hr/report/hr_leave_attendance_report_calendar.py

```diff
@@ -1,11 +1,62 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-from datetime import date
-
 from odoo import api, fields, models, tools, SUPERUSER_ID, _
-
 from odoo.addons.base.models.res_partner import _tz_get
 
+from datetime import date, datetime
+from dateutil.relativedelta import relativedelta
+
+MONTHS = ("Enero",
```

Comment:
Lo que declares fuera de la clase, lo sentencias a que no pueda ser heredado en caso de que se quisiera modificar esa lista, podria ser un metodo que te devuelva esta lista y asi queda totalmente reutilizable en futuros modulos??

Reply:
Es que es solo para algo especifico, si te fijas son los valores que son. No creo que se reutilice en otros módulos por ahora.

Reply:
si en un futuro se necesitara cambiar estos valores, esto creo que seria un handicap, mejor precaver, dont you think?? jejejej